libcxx/include/string

```diff
@@ -2315,10 +2315,13 @@ basic_string<_CharT, _Traits, _Allocator>::__grow_by_and_replace
     size_type __ms = max_size();
```

Comment:
Not attached to this line. Can you folks provide some insight w.r.t. the benefits of doing this in practice? I'm sure you didn't just wake up one morning and decide that this was probably going to be better. How much impact does this have in practice?